<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>-1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
font-size: 14px;
color: while;
background-color:  rgb(38, 41, 42);
}

QLineEdit {
padding-left: 10px;
border-radius: 10px;
color: white;
border: None;
background-color:  rgb(49, 52, 54);
}

QPushButton {
border-radius: 10px;
color: white;
border: None;
background-color: rgb(49, 52, 54);
}

QPushButton:hover {
background-color: #666;
}

QPushButton:pressed {
background-color: #888;
}

QTreeView {
color: white;
border: None;
}



QHeaderView::section { 
border-radius: 10px;
background-color:  rgb(49, 52, 54);
color: white;
border: None;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QFrame" name="frame_3">
        <property name="maximumSize">
         <size>
          <width>176</width>
          <height>60</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>1</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QPushButton" name="pushButtonUps">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color: white;
	border: none;
	background-color: none;
}

QPushButton:hover {
	background-color: #666;
}

QPushButton:pressed {
	background-color: #888;
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="rc_design_pack.qrc">
             <normaloff>:/icons/icons/arrowUps.png</normaloff>:/icons/icons/arrowUps.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QPushButton" name="pushButtonFor">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	icon: none;
    color: white;
	border: none;
	background-color: none;
}

QPushButton:hover {
	background-color: none;

}

QPushButton:pressed {
	background-color: none;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="rc_design_pack.qrc">
             <normaloff>:/icons/icons/arrowFor.png</normaloff>:/icons/icons/arrowFor.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QPushButton" name="pushButtonBack">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	icon: none;
    color: white;
	border: none;
	background-color: none;
}

QPushButton:hover {
	background-color: none;

}

QPushButton:pressed {
	background-color: none;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="rc_design_pack.qrc">
             <normaloff>:/icons/icons/arrowBack.png</normaloff>:/icons/icons/arrowBack.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <item alignment="Qt::AlignTop">
         <widget class="QWidget" name="widget_2" native="true">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	text-align: left;
}</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>3</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>25</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                 <stylestrategy>PreferDefault</stylestrategy>
                </font>
               </property>
               <property name="text">
                <string>Этот компьютер</string>
               </property>
               <property name="icon">
                <iconset resource="rc_design_pack.qrc">
                 <normaloff>:/icons/icons/comp.ico</normaloff>:/icons/icons/comp.ico</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_8">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>25</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                 <stylestrategy>PreferDefault</stylestrategy>
                </font>
               </property>
               <property name="text">
                <string>Видео</string>
               </property>
               <property name="icon">
                <iconset resource="rc_design_pack.qrc">
                 <normaloff>:/icons/icons/video.ico</normaloff>:/icons/icons/video.ico</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_7">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>25</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                 <stylestrategy>PreferDefault</stylestrategy>
                </font>
               </property>
               <property name="text">
                <string>Документы</string>
               </property>
               <property name="icon">
                <iconset resource="rc_design_pack.qrc">
                 <normaloff>:/icons/icons/docs.ico</normaloff>:/icons/icons/docs.ico</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_6">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>25</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                 <stylestrategy>PreferDefault</stylestrategy>
                </font>
               </property>
               <property name="text">
                <string>Загрузки</string>
               </property>
               <property name="icon">
                <iconset resource="rc_design_pack.qrc">
                 <normaloff>:/icons/icons/download.ico</normaloff>:/icons/icons/download.ico</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_5">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>25</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                 <stylestrategy>PreferDefault</stylestrategy>
                </font>
               </property>
               <property name="text">
                <string>Изобрежение</string>
               </property>
               <property name="icon">
                <iconset resource="rc_design_pack.qrc">
                 <normaloff>:/icons/icons/Pictures.ico</normaloff>:/icons/icons/Pictures.ico</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_4">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>25</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                 <stylestrategy>PreferDefault</stylestrategy>
                </font>
               </property>
               <property name="text">
                <string>Музыка</string>
               </property>
               <property name="icon">
                <iconset resource="rc_design_pack.qrc">
                 <normaloff>:/icons/icons/music.ico</normaloff>:/icons/icons/music.ico</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>25</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                 <stylestrategy>PreferDefault</stylestrategy>
                </font>
               </property>
               <property name="text">
                <string>Рабочий стол</string>
               </property>
               <property name="icon">
                <iconset resource="rc_design_pack.qrc">
                 <normaloff>:/icons/icons/desctop.ico</normaloff>:/icons/icons/desctop.ico</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>25</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                 <stylestrategy>PreferDefault</stylestrategy>
                </font>
               </property>
               <property name="text">
                <string>Локальный диск (C:)</string>
               </property>
               <property name="icon">
                <iconset resource="rc_design_pack.qrc">
                 <normaloff>:/icons/icons/sysdisk.ico</normaloff>:/icons/icons/sysdisk.ico</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>30</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="2">
       <widget class="QTreeView" name="treeView">
        <property name="styleSheet">
         <string notr="true">QTreeView::branch {
    color: white;
}
QTreeView::branch:open {
    image: url(./ui/icons/branch12.png);
}
QTreeView::branch:closed:has-children {
    image: url(branch-closed.png);
}</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>lineEdit</zorder>
     <zorder>treeView</zorder>
     <zorder>frame_3</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rc_design_pack.qrc"/>
 </resources>
 <connections/>
</ui>
